tests/test_addon_detail.py

```diff
@@ -495,3 +495,81 @@ def test_click_addon_in_more_addons_by_author(selenium, base_url, variables):
     addon.same_author_addons.addons_by_author_results_items[0].click()
     addon.wait_for_page_to_load()
     assert result_name in addon.name
+
+
+@pytest.mark.nondestructive
+def test_addon_description(selenium, base_url, variables):
+    extension = variables['detail_extension_slug']
+    selenium.get(f'{base_url}/addon/{extension}')
+    addon = Detail(selenium, base_url).wait_for_page_to_load()
+    assert 'About this extension' in addon.description.addon_description_header
+    assert addon.description.addon_description_text.is_displayed()
+
+
+@pytest.mark.nondestructive
+def test_addon_ratings_card(selenium, base_url, variables):
+    extension = variables['detail_extension_slug']
+    selenium.get('{}/addon/{}'.format(base_url, extension))
+    addon = Detail(selenium, base_url).wait_for_page_to_load()
+    assert 'Rate your experience' in addon.ratings.ratings_card_header
+    assert (
+        'How are you enjoying Ghostery - Privacy Ad Blocker'
+        in addon.ratings.ratings_card_summary
+    )
+    # checks that the login button is present in the ratings card
+    # when the add-on detail page is viewed by unauthenticated users
+    addon.ratings.rating_login_button.is_displayed()
+
+
+@pytest.mark.nondestructive
+def test_addon_recommendations(selenium, base_url, variables):
+    extension = variables['detail_extension_slug']
+    selenium.get(f'{base_url}/addon/{extension}')
+    addon = Detail(selenium, base_url).wait_for_page_to_load()
+    recommendations = addon.recommendations.addons_recommendations_results_list
+    # verifies that the recommendations card shows up to 4 recommendations if available
+    assert len(recommendations) <= 4
+
+
+@pytest.mark.nondestructive
+def test_click_addon_recommendations(selenium, base_url, variables):
+    extension = variables['detail_extension_slug']
+    selenium.get(f'{base_url}/addon/{extension}')
+    addon = Detail(selenium, base_url).wait_for_page_to_load()
+    recommendation_name = addon.recommendations.recommendations_results_item[0].text
+    # clicks on a recommendations and checks that the addon detail page is loaded
+    addon.recommendations.recommendations_results_item[0].click()
+    addon.wait_for_page_to_load()
```

Comment:
So this waits for the same page to be loaded? Or a new page

Reply:
This waits for a new add-on page to be loaded - the add-on accessed from the recommendations card.

Reply:
that new page is still a detail page?

Reply:
@jrbenny35 yes, it is another detail page